examples/chip-tool/gen/CHIPClientCallbacks.cpp

```diff
@@ -2004,6 +2004,43 @@ bool emberAfNetworkCommissioningClusterUpdateWiFiNetworkResponseCallback(chip::a
     return true;
 }
 
+bool emberAfOperationalCredentialsClusterOpCSRResponseCallback(chip::app::Command * commandObj, chip::ByteSpan CSR,
+                                                               chip::ByteSpan CSRNonce, chip::ByteSpan VendorReserved1,
+                                                               chip::ByteSpan VendorReserved2, chip::ByteSpan VendorReserved3,
+                                                               chip::ByteSpan Signature)
+{
+    ChipLogProgress(Zcl, "OpCSRResponse:");
+    ChipLogProgress(Zcl, "  CSR: %s", CSR);
+    ChipLogProgress(Zcl, "  CSRNonce: %s", CSRNonce);
+    ChipLogProgress(Zcl, "  VendorReserved1: %s", VendorReserved1);
+    ChipLogProgress(Zcl, "  VendorReserved2: %s", VendorReserved2);
+    ChipLogProgress(Zcl, "  VendorReserved3: %s", VendorReserved3);
+    ChipLogProgress(Zcl, "  Signature: %s", Signature);
```

Comment:
None of these are human readable strings. Is there a reason they are printed as strings here? How does a `chip::ByteSpan` print? I could not find the path used to convert it to a `const char*`

Reply:
It has mostly been used for debugging. I will remove that in a followup, likely just printing the length of the `chip::ByteSpan` for `OCTET_STRING`,  while still printing the content for `CHAR_STRING` (which are `chip::ByteSpan` under the hood).

Reply:
SO what will happen here today, though?

Reply:
I did replace the code that uses `%s` in order to just prints the size and removes a few other places where `chip::Bytespan` was printed badly.

 As discussed I will use https://github.com/project-chip/connectedhomeip/blob/4f0ade5fcd50f99f53188ec401e18c2a22f47e14/src/lib/support/BytesToHex.h#L73 in a followup in order to facilitate debugging. I was originally hoping to do it into this PR but I got distracted by a few other things and since `chip-tool` is broken without this patch I guess it is more urgent to repair it...